dart/lib/src/open_location_code.dart

```diff
@@ -259,7 +262,9 @@ CodeArea decode(String code) {
   if (code.length <= pairCodeLength) {
     return codeArea;
   }
-  var gridArea = decodeGrid(code.substring(pairCodeLength));
+  var _maxDigitCount =
```

Comment:
![image](https://user-images.githubusercontent.com/28839051/50143073-53836680-02b4-11e9-8320-bd4c323b8bbc.png)

Reply:
This can be simplified by changing line 268 to be:

var gridArea = decodeGrid(code.substring(pairCodeLength, min(code.length, maxDigitCount));

Reply:
Thanks!